SQL/contact/get_all.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM contacts
```

Comment:
Why would you want to fetch millions of contacts belonging to all of your users? ;)